### DISCUSSION TITLE
Multiple doc websites?

### DISCUSSION THREAD

**Discussion by bew** [z-shell/zi#144] [Q&A] [answered]
I may be mistaken but there seems to be 2 URLs for the documentation:
- https://z-shell.pages.dev
- https://z.digitalclouds.dev

you seem to post links to one or the other on your twitter account, the 2nd one is used in the about section of this repo, but the first one is used in the about section of https://github.com/zdharma/zi 's repo..

They seems to have the same content.

I think it would help to have a single URL instead of two, no?

**Comment by ss-o** [comment, marked as answer]
Generally, both URL is exactly the same docs.

> I may be mistaken but there seems to be 2 URLs for the documentation:
>
>    https://z-shell.pages.dev/
>    https://z.digitalclouds.dev/

The URL https://z.digitalclouds.dev/ will be used as default as it belongs to us, and the https://z-shell.pages.dev/ URL belongs to Cloudflare. E.g: It would be https://z-shell.github.io if hosted on GitHub. 

> It takes time to update it everywhere :upside_down_face:
